webpack.config.js

```diff
@@ -9,7 +11,8 @@ module.exports = {
   resolve: {
     alias: {
       "devtools": "ff-devtools-libs",
-      "sdk": "ff-devtools-libs/sdk"
+      "sdk": "ff-devtools-libs/sdk",
+      "themes": environment.geckoPath + "devtools/client/themes"
```

Comment:
I'd say either make it optional (somehow conditionally pull in themes), or we should just copy themes into `ff-devtools-libs`. Those files are super easy to copy and don't require any changes, so will be easy to stay up-to-date. I don't love the idea of depending on gecko because I'd like to eventually send this around to people to feedback, and it would be great if they could just fire it up.